lint/csv_format.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import csv
+from typing import Optional, Sequence
+
+import gamla
+
+
+def _format_file(filename):
+    with open(filename, mode="rb") as file_processed:
+        content_before = file_processed.readlines()
+
+    with open(filename, mode="r") as file_processed:
+        rows = tuple(csv.reader(file_processed))
+
+    with open(filename, mode="w") as file_processed:
+        csv.writer(file_processed).writerows(
+            gamla.pipe(
+                rows, gamla.filter(gamla.identity), gamla.sort_by(gamla.head), tuple
```

Comment:
This will only sort by the first char.
so "aa" and "ab" will not be sorted.
Consider taking more letters using `gamla.take`. I guess 3 is enough

Reply:
`rows` is a list of lists, so `gamla.head` here is the first cell of the row. So it will sort each row by its first value, not its first char.

Reply:
ah, got it :)